.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,29 @@
+name: Bug Report
+description: Create a report to help reproduce and fix the issue
+title: "Bug: "
+labels: ["bug"]
+body:
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Reproduction
+      description: How did you manage to make the error happen?
+      placeholder: How did you do it?
+      value: "1. Run StrongDMM v2.0.0.alpha"
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
```

Comment:
We can give information about where to find those logs. Menu: `Help -> Open Logs Folder`.